src/module/actor/sheet/base.js

```diff
@@ -821,6 +821,7 @@ export class ActorSheetSFRPG extends ActorSheet {
         await actorHelper.updateItem(item.id, update);
 
         const itemData = duplicate(item.data);
+        itemData.data = item.data.data; //duplicate doesn't copy .data getter, so we have to graft it on manually.
```

Comment:
This is another instance where we'll need a version check before handling this block. The problem is that you're making your code changes in a v10 world, where this comment is true, but it won't be true on v9. So, we have to check what version of foundry is running, and then populate the data items accordingly.